### PR TITLE
remove leading-none

### LINKED ISSUE #8231: [CAL-1474] remove leading-none
CleanShot 2023-04-12 at 13 26 41@2x

<sub>[CAL-1474](https://linear.app/calcom/issue/CAL-1474/remove-leading-none)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot 2023-04-12 at 13 29 52@2x

after:

CleanShot 2023-04-12 at 13 30 15@2x

### CODE REVIEW

File: packages/features/form-builder/Components.tsx [hunk 1]
```diff
@@ -324,7 +324,7 @@ export const Components: Record<BookingFieldType, Component> = {
                   <Label>
                     {options[0].label}
                     {!readOnly && optionsInputs[options[0].value]?.required ? (
-                      <span className="text-default ml-1 mb-1 text-sm font-medium leading-none">*</span>
+                      <span className="text-default ml-1 mb-1 text-sm font-medium ">*</span>
```
Comment (PeerRich) [inline packages/features/form-builder/Components.tsx:327]:
```suggestion
                      <span className="text-default ml-1 mb-1 text-sm font-medium">*</span>
```

File: packages/features/shell/Shell.tsx [hunk 2]
```diff
@@ -307,10 +307,10 @@ function UserDropdown({ small }: { small?: boolean }) {
             {!small && (
               <span className="flex flex-grow items-center truncate">
                 <span className="flex-grow truncate text-sm">
-                  <span className="text-emphasis mb-1 block truncate font-medium leading-none">
+                  <span className="text-emphasis mb-1 block truncate font-medium ">
```
Comment (PeerRich) [inline packages/features/shell/Shell.tsx:310]:
```suggestion
                  <span className="text-emphasis mb-1 block truncate font-medium">
```

File: packages/ui/components/card/FormCard.tsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ export default function FormCard({
       <div className="w-full">
         <div className="flex items-center justify-between">
           <div>
-            <span className="text-emphasis text-sm font-semibold leading-none">{label}</span>
+            <span className="text-emphasis text-sm font-semibold ">{label}</span>
```
Comment (PeerRich) [inline packages/ui/components/card/FormCard.tsx:57]:
```suggestion
            <span className="text-emphasis text-sm font-semibold">{label}</span>
```

**Review by PeerRich — COMMENTED** [review]
fixed some lint

File: packages/features/shell/Shell.tsx [hunk 4]
```diff
@@ -307,10 +307,10 @@ function UserDropdown({ small }: { small?: boolean }) {
             {!small && (
               <span className="flex flex-grow items-center truncate">
                 <span className="flex-grow truncate text-sm">
-                  <span className="text-emphasis mb-1 block truncate font-medium leading-none">
+                  <span className="text-emphasis mb-1 block truncate font-medium">
```
Comment (PeerRich) [inline packages/features/shell/Shell.tsx:310]:
```suggestion
                  <span className="text-emphasis mb-1 block truncate font-medium leading-none">
```

File: packages/features/shell/Shell.tsx [hunk 5]
```diff
@@ -307,10 +307,10 @@ function UserDropdown({ small }: { small?: boolean }) {
...
                     {user.name || "Nameless User"}
                   </span>
-                  <span className="text-default block truncate font-normal leading-none">
+                  <span className="text-default block truncate font-normal">
```
Comment (PeerRich) [inline packages/features/shell/Shell.tsx:313]:
```suggestion
                  <span className="text-default block truncate font-normal leading-none">
```

**Review by sean-brydon — APPROVED** [review]

**Pull request merged**